go2rtc_client/models.py

```diff
@@ -8,6 +8,16 @@
 from mashumaro.mixins.orjson import DataClassORJSONMixin
 
 
+@dataclass
+class ApplicationInfo(DataClassORJSONMixin):
+    """Application info model.
+
+    Currently only the server version is exposed.
+    """
+
+    version: str
```

Comment:
Maybe we should return an `AwesomeVersion` instead of a string?

Reply:
Sounds good.